kuma/javascript/src/article.jsx

```diff
@@ -63,6 +65,21 @@ export default function Article({ document }: DocumentProps) {
                     : 'article text-content'
             }
         >
+            {document.locale !== locale && (
+                <div id="doc-pending-fallback" className="warning">
```

Comment:
```suggestion
                <div className="warning">
```
That ID is not used in CSS. On the [wiki it's used to trigger these lines](https://github.com/mozilla/kuma/blob/59ca3f7d0e508e063239057745c9190a533a6c74/kuma/static/js/wiki.js#L224-L228):
```
mdn.analytics.trackError('Translation Pending', 'displayed');
```
I guess it simply means we need to have that analytics track thing on the React side too.